pyt/formatters/json.py

```diff
@@ -0,0 +1,33 @@
+"""This formatter outputs the issues in JSON."""
```

Comment:
So the formatters directory was heavily [inspired by Bandit](https://github.com/openstack/bandit/tree/master/bandit/formatters).